src/recoil/app-theme/app-theme.state.ts

```diff
@@ -19,22 +20,23 @@ const isWithinRange = (date: DateTime<true> | DateTime<false>) => {
   return now >= date.minus({ days: 13 }) && now <= date.plus({ days: 1 })
 }
 
-const halloween = DateTime.fromObject({ month: 10, day: 31 }).endOf(`day`)
+//const halloween = DateTime.fromObject({ month: 10, day: 31 }).endOf(`day`)
+const christmas = DateTime.fromObject({ month: 12, day: 25 }).endOf(`day`)
 const defaultTheme = ((): AppTheme => {
   // TODO: The following themes will be implemented in the future:
   // if (isWithinRange(newYears)) return AppTheme.NewYear // Happy New Year theme
   // if (isWithinRange(valentines)) return AppTheme.Valentines // Love theme
   // if (isWithinRange(spring)) return AppTheme.Spring // Cherry blossom theme
   // if (isWithinRange(summer)) return AppTheme.Summer // Beach theme
-  if (isWithinRange(halloween)) return AppTheme.Halloween
-  // if (isWithinRange(christmas)) return AppTheme.Chrstimas // Christmas theme
+  // if (isWithinRange(halloween)) return AppTheme.Halloween
+  if (isWithinRange(christmas)) return AppTheme.Christmas // Christmas theme
 
   // Default:
   return AppTheme.Basic
 })()
 
 export const appThemeState = atom<AppTheme>({
   key: Rkp.AppTheme,
-  default: defaultTheme,
-  // default: AppTheme.Halloween, // You can always override the default theme here, and compile will fail if you forget to remove it
+  //default: defaultTheme,
+  default: AppTheme.Christmas, // You can always override the default theme here, and compile will fail if you forget to remove it
```

Comment:
Back to original setting before PR please!
```suggestion
  default: defaultTheme,
  // default: AppTheme.Christmas, // You can always override the default theme here, and compile will fail if you forget to remove it
```